htdocs/js/ui/discovery_page.js

```diff
@@ -35,7 +35,7 @@ RCloud.UI.discovery_page = (function() {
                                     last_commit: new Date(current.last_commit).toDateString(),
                                     username: current.username,
                                     num_stars: editor.num_stars(current[0]),
-                                    image_src: "data:image/png;base64," + thumb_src,
+                                    image_src: thumb_src,
```

Comment:
oh, i see, you added it to `rcloud.set.thumb` - that's reasonable but i'm going to keep that as raw data and assemble the url in the above line instead.